pex/vendor/__init__.py

```diff
@@ -20,7 +20,8 @@ def _root():
   return path
 
 
-class VendorSpec(collections.namedtuple('VendorSpec', ['key', 'requirement', 'rewrite'])):
+class VendorSpec(collections.namedtuple('VendorSpec',
+                                        ['key', 'requirement', 'rewrite', 'constrain'])):
```

Comment:
Can you please add `constrain` to the docstring? It took me a moment to realize this is a bool and I thought it was a typo of `constraint`

Reply:
Done.